components/schemes/Typography/tests/unit/MDCTypographySchemeTests.m

```diff
@@ -44,4 +44,26 @@ - (void)testInitializer {
   XCTAssertEqual(latestScheme.overline, scheme201804.overline);
 }
 
+- (void)testTypographySchemeCopyTest {
+  // Given
+  MDCTypographyScheme *typographyScheme = [[MDCTypographyScheme alloc] init];
+  MDCTypographyScheme *typographySchemeCopy = [typographyScheme copy];
```

Comment:
This is the `// When` statement.  Please mark it with a comment.

Reply:
Done.